.github/workflows/_integration-tests-api.yml

```diff
@@ -40,6 +40,7 @@ jobs:
           -f api/docker/app/Dockerfile \
           --cache-from=type=local,src=/tmp/.buildx-cache \
           --cache-to=type=local,dest=/tmp/.buildx-cache-new \
+          --build-arg ENVIRONMENT=local \
```

Comment:
These are for the parameters.yml to use localstack as we run these tests locally